src/deployment/registration.py

```diff
@@ -646,33 +646,34 @@ def set_app_audience(
         raise Exception(err_str)
 
 
-def add_user(objectId: str, principalId: str) -> None:
+def add_users(objectId: str, principalIds: List[str]) -> None:
```

Comment:
i would keep the function as is and call it in a loop instead

Reply:
Can do!